libgalois/include/katana/PropertyGraph.h

```diff
@@ -251,19 +251,15 @@ class KATANA_EXPORT PropertyGraph {
     auto pm = rdg_.part_metadata();
     return topology_.nodes(0, pm.num_owned_);
   }
-  GraphTopology::nodes_range outgoing_mirrors() const {
+  GraphTopology::nodes_range mirrors() const {
     auto pm = rdg_.part_metadata();
-    return topology_.nodes(pm.num_owned_, pm.num_nodes_with_edges_);
-  }
-  GraphTopology::nodes_range incoming_mirrors() const {
-    auto pm = rdg_.part_metadata();
-    return topology_.nodes(pm.num_nodes_with_edges_, pm.num_nodes_);
+    return topology_.nodes(pm.num_owned_, pm.num_nodes_);
```

Comment:
I assume you know better than I but just to confirm: this doesn't change when there are completely disconnected nodes?

Reply:
I didn't understand your question. Isolated (completely disconnnected) nodes can be a master (i.e., in the first `num_owned_` nodes) but they cannot be a mirror because we create mirrors only for edges.

Reply:
Ah yep it makes so much sense when you say it like that 👍